GameTicTacToe/src/StageTwoDemo.java

```diff
@@ -0,0 +1,22 @@
+import java.util.Scanner;
+
+public class StageTwoDemo {
+    public static void main(String[] args) {
+        String game[][] = new String[3][3];
```

Comment:
@maria-salmasian I think you cane make a some kind of object instead of use String array